src/test/debuginfo/vec-slices.rs

```diff
@@ -53,6 +53,11 @@
 // gdb-command:print *((int64_t[2]*)('vec_slices::MUT_VECT_SLICE'.data_ptr))
 // gdb-check:$15 = {64, 65}
 
+//gdb-command:print mut_slice.length
```

Comment:
The second one casts it to an array and tries to print it. That relies on the C pretty printer, not ten Rust one. `print slice` and `print mut_slice` should be what we are testing. I don't see the value in the array-cast test for mut slices.